lib/companies/pending_changes.ex

```diff
@@ -9,32 +9,15 @@ defmodule Companies.PendingChanges do
   alias Companies.Schema.{Company, Job, Industry, PendingChange}
 
   def all do
-    query =
-      from p in PendingChange,
-        select: p,
-        where: p.approved == false
+    query = from p in PendingChange, where: p.approved == false
 
     Repo.all(query)
   end
 
-  def approve(change_id) do
-    with %{action: action, changes: changes, resource: resource} = change <- Repo.get!(PendingChange, change_id),
-         module <- resource_module(resource),
-         changeset <- changeset(module, changes),
-         {:ok, _changes} <- apply_changes(action, changeset) do
-      change
-      |> PendingChange.changeset(%{approved: true})
-      |> Repo.update()
-    else
-      nil -> {:error, "change not found"}
-      {:error, changeset} -> {:error, changeset}
-    end
+  def get!(id) do
```

Comment:
Do we really need to define this? We already have `get/1` and this isn't going to widely consume. I think a smaller API makes testing things easier and reduces the number of potential code paths

Reply:
get/1 is doing some other things. Granted, this should be a private method and not parted of the API.